http/src/main/resources/reference.conf

```diff
@@ -23,11 +23,15 @@ dataproc {
   #vpcSubnet = "default"
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-2-79-debian9-9ba5905"
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-4-15-debian9-9ba5905"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-1-4-15-debian9-hhhhhhh"
```

Comment:
I've appended `dataproc/gce` to the image base name (which used to be `custom-leo-image`) to differentiate the two. 

Was planning on updating `hhhhhhh` with the hash that's in `develop` at the time of the PR approval but wondering if there's a different type of process for this sort of thing 🤔

Reply:
`legacyCustomDataprocImage` should probably be updated with the new name/hash as well?

Reply:
Also this is likely causing automation tests to fail. Maybe you could manually generate some Dataproc+GCE images using your new CI, and update `reference.conf` to include those hashes?

Reply:
> Even though it's legacy it's continually being rebuilt alongside the other dataproc image because VM images expire after 60 days.

Ah forgot about that, thanks!